docs/source/model-registry.rst

```diff
@@ -188,8 +190,10 @@ You can transition a registered model in the registry to one of the stages: **St
         stage="production"
     )
 
-Listing and Searching Models
-----------------------------
+The accepted values for <stage> are: "Staging"|"staging", "Archived"|"archived", "Production"|"production", "None"|"none".
```

Comment:
Smart quotes causes this to look a bit weird, and having both the capitalized and non- versions listed seems a bit odd. What do you think?

![image](https://user-images.githubusercontent.com/1400247/79385502-8d529480-7f1d-11ea-97b3-99a6e0fb9907.png)

Reply:
I just made it consistent with https://docs.databricks.com/applications/mlflow/model-registry.html#transition-a-model-version-stage docs. 

We also don't document what value a "stage" can take in our API docs either https://mlflow.org/docs/latest/python_api/mlflow.tracking.html#mlflow.tracking.MlflowClient.transition_model_version_stage

For now, having it here until we fix the APIs might be good. I don't feel strongly either way